utils/file_backend_s3_test.go

```diff
@@ -13,20 +13,20 @@ func TestCheckMandatoryS3Fields(t *testing.T) {
 	cfg := model.FileSettings{}
 
 	err := CheckMandatoryS3Fields(&cfg)
-	if err == nil || err.Message != "api.admin.test_s3.missing_s3_bucket" {
+	if err != nil && err.Message != "api.admin.test_s3.missing_s3_bucket" {
```

Comment:
This seems like it was right before. Don't we want to fail if `err == nil`?

Reply:
correct. done